tests/integration/smoke-test.json

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
nit: Let's name this file `smoke-tests.json`

Reply:
I've updated this in my latest revision.